src/Components/fe-otp/FeOtp.js

```diff
@@ -37,26 +48,50 @@ export class FeOtp extends LocalizeMixin(LitElement) {
       return;
     }
 
-    const loginData = serializedValue;
     this.dispatchEvent(
```

Comment:
i think event triggered is not needed, bcz your validation here itself, so nothing needed as of now

Reply:
But in test case this event is been used for validation

Reply:
as of now this one is not needed, may be we can send if OTP validation is success, because that will helps parent component to navigate to next screen, in test case, just check whether `checkOTP` function is triggered or not

Reply:
okay

Reply:
complete will trigger only when otp validaiton is success, this way even API call not trigger also this event will trigger

Reply:
I will add the event under checkout function.